policy/modules/system/systemd.if

```diff
@@ -30,8 +30,19 @@ template(`systemd_role_template',`
 		attribute systemd_user_session_type, systemd_log_parse_env_type;
 		type systemd_user_runtime_t, systemd_user_runtime_notify_t;
 		type systemd_run_exec_t, systemd_analyze_exec_t;
+		type systemd_user_home_t, systemd_user_home_unit_t;
+		type systemd_user_runtime_unit_t;
+		class dbus { acquire_svc send_msg };
```

Comment:
these shouldnt be referenced here

Reply:
OK. Is it appropriate to have these in an `optional_policy` block in the template instead?